neo/public/javascripts/common/services.js

```diff
@@ -1,5 +1,39 @@
 angular.module('cloudberry.common', [])
-  .service('Asterix', function($http, $timeout, $location) {
+  .factory('cloudberryConfig', function(){
+    return {
+      normalizationUpscaleFactor: 1000 * 1000,
+      normalizationUpscaleText: "/M",
+      getPopulationTarget: function(parameters){
+        switch (parameters.geoLevel) {
+          case "state":
+            return {
+              joinKey: ["state"],
+              dataset: "twitter.dsStatePopulation",
+              lookupKey: ["stateID"],
+              select: ["population"],
+              as: ["population"]
+            };
+          case "county":
+            return {
+              joinKey: ["county"],
+              dataset: "twitter.dsCountyPopulation",
+              lookupKey: ["countyID"],
+              select: ["population"],
+              as: ["population"]
+            };
+          case "city":
+            return {
+              joinKey: ["city"],
+              dataset: "twitter.dsCityPopulation",
+              lookupKey: ["cityID"],
+              select: ["population"],
+              as: ["population"]
+            };
+        }
+      }
+    };
+  })
+  .service('Asterix', function($http, $timeout, $location, cloudberryConfig) {
```

Comment:
could you take this chance to rename this `Asterix`? it's been a while. It should be `cloudberry`.